src/makeCalculator.js

```diff
@@ -3,8 +3,44 @@
 /**
  * @return {object}
  */
-function makeCalculator() {
-  // write code here
+function makeCalculator(operate) {
+  const obj = {
+    result: 0,
+
+    reset: function () {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate: function (callback, number) {
+      this.result = callback(this.result, number);
+
+      return this;
+    },
+
+    add: function (currentResult, number) {
+      return currentResult + number;
+    },
+
+    subtract: function (currentResult, number) {
+      return currentResult - number;
```

Comment:
Use the method shorthand syntax for defining methods in an object. Replace 'subtract: function (currentResult, number) {' with 'subtract(currentResult, number) {'.